scenarios/tracking/README.md

```diff
@@ -2,4 +2,131 @@
 
 ```diff
 + June 2020: This work is ongoing.
-```
\ No newline at end of file
+```
+
+This directory provides examples and best practices for building multi-object tracking systems. Our goal is to enable the users to bring their own datasets and train a high-accuracy model easily. While there are many open-source trackers available, we have implemented the [FairMOT tracker](https://github.com/ifzhang/FairMOT) specifically, as its algorithn has shown the best tracking performance in recent MOT benchmarking challenges, at fast inference speed.
+
+## Technology
+Multi-object-tracking (MOT) is one of the hot research topics in Computer Vision, due to its wide applications in autonomous driving, traffic surveillance, etc. It builds on object detection technology, in order to detect and track all objects in a dynamic scene over time, without any prior knowledge about the appearance and number of target objects. Inferring target trajectories correctly across successive image frames remains challenging: occlusion happens when objects overlap; the number of and appearance of objects can change. Compared to object detection algorithms, which aim to output rectangular bounding boxes around the objects, MOT algorithms additionally associated an ID number to each box to identify that specific object across the image frames. 
+
+As seen in the figure below ([Ciaparrone, 2019](https://arxiv.org/pdf/1907.12740.pdf)), a typical multi-object-tracking algorithm performs part or all of the following steps:
+* Detection: Given the input raw image frames (step 1), the detector identifies object(s) on each image frame as bounding box(es) (step 2).
+* Feature extraction/motion prediction: For every detected object, visual appearance and motion features are extracted (step 3). Sometimes, a motion predictor (e.g. Kalman Filter) is also added to predict the next position of each tracked target. 
+* Affinity: The feature and motion predictions are used to calculate similarity/distance scores between pairs of detections and/or tracklets, or the probabilities of detections belonging to a given target or tracklet (step 4). 
+* Association: Based on these scores/probabilities, a specific numerical ID is assigned to each detected object as it is tracked across successive image frames (step 5).
+
+<p align="center">
+<img src="./media/figure_motmodules.jpg" width="600" align="center"/>
+</p> 
+
+
+**Online vs offline(batch tracking)**   
```

Comment:
These sections are good but a bit too detailed. I'd suggest moving most sections from here until including "evaluation metrics" to the FAQ. Consider summarizing the main points into high-level sentences, but do keep the tables :) A bit like here perhaps.
https://github.com/microsoft/computervision-recipes/tree/master/scenarios/action_recognition